kubernetes/client/doc.go

```diff
@@ -0,0 +1,91 @@
+// Copyright © 2018 Genome Research Limited
+// Author: Theo Barber-Bany <tb15@sanger.ac.uk>.
+//
+//  This file is part of wr.
+//
+//  wr is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU Lesser General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  wr is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU Lesser General Public License for more details.
+//
+//  You should have received a copy of the GNU Lesser General Public License
+//  along with wr. If not, see <http://www.gnu.org/licenses/>.
+
+package client
+
+/*
+
+Package client provides functions to interact with a kubernetes cluster, used to
+create resources so that you can spawn runners, then delete those
```

Comment:
Is this library independent of wr and the manager (ideally it should be)? If so, can you explain things in more general terms, without reference to wr?

Even if it is tied in to how wr works, could you give a higher-level "architectural" overview of what this library does and how, right here at the top?

A key question to answer in the first paragraph is "why would I want to use this library?", which involves answering "what does this give me over just using github.com/kubernetes/client-go directly?".